doc/requirements.rst

```diff
@@ -56,35 +45,39 @@ Motor and PyMongo
+| 2.2               | 3.11+           |
++-------------------+-----------------+
 
 Motor and MongoDB
 `````````````````
 
-+---------------------------------------------------------------------------+
-|                    MongoDB Version                                        |
-+=====================+=====+=====+=====+=====+=====+=====+=====+=====+=====+
-|                     | 2.2 | 2.4 | 2.6 | 3.0 | 3.2 | 3.4 | 3.6 | 4.0 | 4.2 |
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-| Motor Version | 1.0 |  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-|               | 1.1 |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-|               | 1.2 |**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-|               | 1.3 |**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-|               | 2.0 |**N**|**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
-|               | 2.1 |**N**|**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |
-+---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
++---------------------------------------------------------------------------+-----+
+|                                  MongoDB Version                                |
++=====================+=====+=====+=====+=====+=====+=====+=====+=====+=====+=====+
+|                     | 2.2 | 2.4 | 2.6 | 3.0 | 3.2 | 3.4 | 3.6 | 4.0 | 4.2 | 4.4 |
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+| Motor Version | 1.0 |  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|**N**|**N**|**N**|
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+|               | 1.1 |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|**N**|**N**|
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+|               | 1.2 |**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|**N**|
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+|               | 1.3 |**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |**N**|**N**|**N**|
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+|               | 2.0 |**N**|**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |**N**|
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+|               | 2.1 |**N**|**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |**N**|
++---------------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+
+|               | 2.2 |**N**|**N**|**N**|  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |
```

Comment:
Pymongo still supports MongoDB 2.6 so I think motor (2.X) should still support it as well.

Reply:
I believe officially support was dropped after we stopped testing Motor explicitly against MongoDB 2.6. While I agree with your rationale, I am not particularly excited about re-introducing 2.6 support after 2 versions have already advertised it as being unsupported. Maybe @techbelle can weigh in here with some Motor usage numbers by MongoDB version - I suspect usage skews heavily towards newer server versions for this driver.

Reply:
2.6 was dropped in https://jira.mongodb.org/browse/MOTOR-252.